spec/features/user_registers_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'spec_helper'
+
+feature 'user register a new account', { js: true, vcr: true } do
```

Comment:
I've tried, then got this warning message.

*****************************************************************
WARNING: You have passed symbols ([:js, :vcr]) as metadata
arguments to a configuration option.

In RSpec 3, these symbols will be treated as metadata keys with
a value of `true`.  To get this behavior now (and prevent this
warning), you can set a configuration option:

RSpec.configure do |c|
  c.treat_symbols_as_metadata_keys_with_true_values = true
end

Note that this config setting should go before your other config
settings so that they can use symbols as metadata.
*****************************************************************